homeassistant/components/sensor/upnp.py

```diff
@@ -0,0 +1,78 @@
+"""
+Support for UPnP Sensors (IGD).
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.upnp/
+"""
+import asyncio
+import logging
+
+from homeassistant.components.upnp import DATA_UPNP, UNITS
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+""" sensor_type: [friendly_name, convert_unit, icon] """
```

Comment:
Use `#` for comments. Only use `"""` for first line in module/class/method to indicate it's a doc string.